pkg/config/msp_test.go

```diff
@@ -91,50 +440,61 @@ func TestMSPToProtoFailure(t *testing.T) {
 	fabricMSPConfig.SigningIdentity.PrivateSigner.KeyMaterial = &ecdsa.PrivateKey{}
 
 	fabricMSPConfigProto, err := fabricMSPConfig.toProto()
-	gt.Expect(err).To(MatchError("pem encode X.509 private key: marshalling PKCS8 private key: x509: unknown curve while marshaling to PKCS#8"))
+	gt.Expect(err).To(MatchError("pem encode ec private key: marshalling ec private key: x509: unknown elliptic curve"))
 	gt.Expect(fabricMSPConfigProto).To(BeNil())
 }
 
 func baseMSP() MSP {
+	certBlock, _ := pem.Decode([]byte(dummyCert))
+	cert, _ := x509.ParseCertificate(certBlock.Bytes)
+
+	privKeyBlock, _ := pem.Decode([]byte(dummyPrivateKey))
+	privKey, _ := x509.ParsePKCS8PrivateKey(privKeyBlock.Bytes)
+
+	crlBlock, _ := pem.Decode([]byte(dummyCRL))
+	crl, _ := x509.ParseCRL(crlBlock.Bytes)
+
 	return MSP{
-		Name:              "",
-		RootCerts:         []x509.Certificate{},
-		IntermediateCerts: []x509.Certificate{},
-		Admins:            []x509.Certificate{},
-		RevocationList:    []pkix.CertificateList{},
+		Name:              "MSPID",
+		RootCerts:         []x509.Certificate{*cert},
```

Comment:
With all of the back and forth on ref vs. value, I'd expect the cert to be a ref since that's how the stdlib deals with it.

Reply:
I did notice that while working through this that there were a number of places where I had to reference or dereference certain variables. I'll probably throw a story in the backlog to revisit some of the types to change them to what makes sense in the context of how the variables are actually used (ie in this example they might as well be pointers)